python/lsst/sconsUtils/eupsForScons.py

```diff
@@ -38,3 +38,10 @@ def setupEnvNameFor(productName):
     
     utils.setupEnvNameFor = setupEnvNameFor
 
+def getEups():
+    """ Return a cached Eups instance, auto-creating if necessary """
+    try:
+        return getEups._eups
+    except:
```

Comment:
Agreed. Will fix.

I saw the idiom w/o the exception spec elsewhere in builders.ProductDir(), and thought that was the desired convention.